kuma/javascript/src/payments/subscription-form.jsx

```diff
@@ -1,76 +1,181 @@
 // @flow
 import * as React from 'react';
-import { useRef, useState } from 'react';
+import { useContext, useEffect, useRef, useState } from 'react';
 
 import { getLocale, gettext, Interpolated } from '../l10n.js';
+import UserProvider from '../user-provider.jsx';
+import { getCookie } from '../utils';
+
+const SUBSCRIPTION_URL = '/api/v1/subscriptions';
+
+/**
+ * Loads the script given by the URL and cleans up after itself
+ * @returns {(null | Promise)} Indicating whether the script has successfully loaded
+ */
+function useScriptLoading(url) {
+    const [loadingPromise, setLoadingPromise] = useState<null | Promise<void>>(
+        null
+    );
+    useEffect(() => {
+        let script;
+        if (!loadingPromise) {
+            script = document.createElement('script');
+            setLoadingPromise(
+                new Promise((resolve, reject) => {
+                    script.onload = () => resolve;
```

Comment:
I don't know where my comment went! Gah! Anyway, this breaks as is. But if you change it to
```suggestion
                    script.onload = resolve;
```
...then things work.